### PR TITLE
cmd/clusterlink: Add fabric name to CLI

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add an option to specify the fabric name when creating certificates, by default, will use "default_fabric".

### CODE REVIEW

File: cmd/clusterlink/cmd/create/create_fabric.go [hunk 1]
```diff
@@ -17,32 +17,56 @@ import (
...
 		},
 	}
+
+	opts.AddFlags(cmd.Flags())
+	return cmd
+}
+
+// Run the 'create fabric' subcommand.
+func (o *FabricOptions) Run() error {
+	fabricCert, err := bootstrap.CreateFabricCertificate()
```
Comment (orozery) [inline cmd/clusterlink/cmd/create/create_fabric.go:56]: Perhaps we want to encode the fabric name inside the root CA?

File: cmd/clusterlink/config/config.go [hunk 2]
```diff
@@ -18,10 +18,8 @@ import (
 )
 
 const (
-	// PrivateKeyFileName is the filename used by private key files.
-	PrivateKeyFileName = "key.pem"
-	// CertificateFileName is the filename used by certificate files.
-	CertificateFileName = "cert.pem"
+	// DefaultFabric is the default fabric name.
+	DefaultFabric = "default_fabric"
```
Comment (elevran) [inline cmd/clusterlink/config/config.go:22]: I would drop this default. Either use `default` or leave empty and require a fabric name/generate a random one.

**Review by elevran — APPROVED** [review]

File: cmd/clusterlink/cmd/create/create_peer.go [hunk 3]
```diff
@@ -71,13 +74,13 @@ func (o *PeerOptions) RequiredFlags() []string {
...
 	// save certificate to file
-	err := os.WriteFile(filepath.Join(outDirectory, config.CertificateFileName), cert.RawCert(), 0o600)
+	err := os.WriteFile(filepath.Join(outDirectory, o.Name+".cert"), cert.RawCert(), 0o600)
 	if err != nil {
 		return err
 	}
 
 	// save private key to file
-	return os.WriteFile(filepath.Join(outDirectory, config.PrivateKeyFileName), cert.RawKey(), 0o600)
+	return os.WriteFile(filepath.Join(outDirectory, o.Name+".key"), cert.RawKey(), 0o600)
```
Comment (orozery) [inline cmd/clusterlink/cmd/create/create_peer.go:83]: Why did you change the file names to include the peer name?

Comment (kfirtoledo) [reply cmd/clusterlink/cmd/create/create_peer.go:83]: That is what I wanted to have <peer_name>.cert and <peer_name>.key that describes to whom this key belongs.
